### DISCUSSION TITLE
how to setup the app

### DISCUSSION THREAD

**Discussion by threefive6** [JoshLudahl/Warbler-Weather#207] [Q&A]
bit confused on how to set up the app 

it says WEATHER_BASE_URL is not found, even after I put WEATHER_API_KEY=[whatever]. i tried the url https://api.openweathermap.org/data/2.5/
but it says 'Error, please try again with an internet connection' when running it through an emulator

let me know what to do, thanks

**Comment by JoshLudahl** [comment]
A bit late, but the API uses the newer version, so 2.5 won't work. You would need to sign up for the paid tier in order to have a working API version. Their free tier should give you enough tokens to run it for a few users.
